analysis/src/analyzer.js

```diff
@@ -61,6 +61,11 @@ class AnalyzerRunner {
       const rootUrl = await analyzer.urlResolver.resolve('');
       const isInPackage = feature => feature.sourceRange &&
         feature.sourceRange.file.startsWith(rootUrl);
+      // Filter out test files. This matches both `/test/abc.html` and
+      // `/my-test.html`.
+      const testRegex = /\btests?\b/;
```

Comment:
This seems pretty aggressive -- any thing with the word test in it at all. E.g. this would break the test-fixture element I assume. Maybe just filtering out a top-level test (and demo?) directory is enough?

Reply:
Done. PTAL @aomarks